app/src/main/java/de/tum/in/tumcampusapp/component/ui/ticket/EventsController.java

```diff
@@ -55,21 +60,34 @@ public void downloadFromService(boolean force) {
             // NOTE: the dummy data on the server contains a dummy ticket for event id 2
             //       thus, we add another event with id 2 for testing purposes
             // TODO: remove this when dummy data on server is made consistent or real data is used!
-            Event event = events.get(0);
+            /*Event event = events.get(0);
+
             events.add(new Event(2,event.getImage(), event.getTitle() + "2",
                     event.getDescription() + " Not to say extremely nice!",
                     event.getLocality(), event.getDate(), event.getLink()));
-
+            */
```

Comment:
Just remove this code already. I've deleted all bought tickets, so there are no tickets present currently.

Reply:
removed it, the EventsController still needs some changes though (in other issue).